tests/cpp/include/test_mkldnn.h

```diff
@@ -189,8 +202,10 @@ inline static TestArrayShapes GetTestArrayShapes() {
     std::vector<mkldnn::memory::format> formats = GetMKLDNNFormat(4, dtype);
     pds.push_back(GetMemPD(s1, dtype, formats[0]));
     pds.push_back(GetMemPD(s2, dtype, formats[1]));
+    */
```

Comment:
if you comment the code out, it's better just to remove it.

Reply:
It's for debug. Has been reverted.